src/scripts/recon/allowed-methods/__main__.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+from requests import request
+from requests.exceptions import *
+
+from src.core.base.recon import ReconRunner, PossibleKeys
+from src.core.utils.response import ScriptResponse
+from src.core.utils.validators import validate_kwargs
+
+
+class Defaults:
+    # List of the Request Methods Recognized by Apache
```

Comment:
Wow, that's good! Also, please add one random method, like, something that is impossible, maybe a method like "EWQEJKQLJ" or you can generate it by yourself dynamically, just to check that server is not responding to not existing methods.